device-types/Juniper/SRX2300.yaml

```diff
@@ -0,0 +1,75 @@
+---
+manufacturer: Juniper
+model: SRX2300
+slug: juniper-srx2300
+part_number: SRX2300
+front_image: true
+rear_image: true
+u_height: 1
+is_full_depth: true
+weight: 8.6
+weight_unit: kg
+airflow: front-to-rear
+comments: '[Juniper SRX2300 Data Sheet (PDF)](https://www.juniper.net/content/dam/www/assets/datasheets/us/en/security/srx2300-firewall-datasheet.pdf)'
+interfaces:
+  - name: fxp0
+    type: 1000base-t
+    mgmt_only: true
+  - name: em0
+    type: 1000base-x-sfp
+  - name: em1
+    type: 1000base-x-sfp
+  - name: mge-0/0/0
+    type: 10gbase-t
+  - name: mge-0/0/1
+    type: 10gbase-t
+  - name: mge-0/0/2
+    type: 10gbase-t
+  - name: mge-0/0/3
+    type: 10gbase-t
+  - name: mge-0/0/4
+    type: 10gbase-t
+  - name: mge-0/0/5
+    type: 10gbase-t
+  - name: mge-0/0/6
+    type: 10gbase-t
+  - name: mge-0/0/7
+    type: 10gbase-t
+  - name: xe-0/1/0
+    type: 10gbase-x-sfpp
+  - name: xe-0/1/1
+    type: 10gbase-x-sfpp
+  - name: xe-0/1/2
+    type: 10gbase-x-sfpp
+  - name: xe-0/1/3
+    type: 10gbase-x-sfpp
+  - name: xe-0/1/4
+    type: 10gbase-x-sfpp
+  - name: xe-0/1/5
+    type: 10gbase-x-sfpp
+  - name: xe-0/1/6
+    type: 10gbase-x-sfpp
+  - name: xe-0/1/7
+    type: 10gbase-x-sfpp
+  - name: et-0/2/0
+    type: 25gbase-x-sfp28
+  - name: et-0/2/1
+    type: 25gbase-x-sfp28
+  - name: et-0/2/2
+    type: 25gbase-x-sfp28
+  - name: et-0/2/3
+    type: 25gbase-x-sfp28
+  - name: et-0/3/0
+    type: 100gbase-x-qsfp28
+  - name: et-0/3/1
+    type: 100gbase-x-qsfp28
+power-ports:
```

Comment:
Thank you for your contribution,

As the PSUs are field replaceable units they should be modeled as `module-bays` then add the required PSU modules if they don't already exist

Reply:
Will do. Thanks.

Reply:
I made the changes you requested.